frontend/src/lib/components/Annotations/annotationsLogic.ts

```diff
@@ -69,7 +76,6 @@ export const annotationsLogic = kea<annotationsLogicType<AnnotationsLogicProps>>
                     date_marker: date_marker,
                     created_at: created_at.toISOString(),
                     updated_at: created_at.toISOString(),
-                    created_by: userLogic.values.user,
```

Comment:
There was an existing bug where we couldn't create any local annotations because kea was throwing an error saying that `userLogic.values.user` couldn't be fetched inside of a reducer.

I went the lazy way and removed it altogether, but seeing that this introduces another bug I'll think of another way to pass through user information in the annotationlogic api

Reply:
Edit: `createAnnotation` is broken and that code path is almost never hit after our refactor to use `short_id` and `insight.id`. Furthermore nothing is being done with `annotationsToCreate` once it is populated, which means that users lose annotations after saving insights even after being told via toast that they will persist when an unsaved insight is added to a dashboard. 

From a user's perspective, the toast is confusing because (1) saving annotations after saving the insight is already expected, and (2) the user doesn't care about what happens to the insight-level annotations if they don't save it. Because of these reasons, I'm going to prune out `createAnnotation` and anything downstream that.